lms/templates/instructor/instructor_dashboard_2/instructor_dashboard_2.html

```diff
@@ -126,3 +126,7 @@ <h2 class="hd hd-2" id="header-${ section_data['section_key'] }">${ section_data
         </main>
   </div>
 </div>
+
+<%static:require_module_async module_name="js/dateutil_factory" class_name="DateUtilFactory">
+    DateUtilFactory.transform(iterationKey=".localized-datetime");
+</%static:require_module_async>
```

Comment:
I added this here to get it further to the front of the line in the async commands. There's a noticeable delay in devstack, and this seemed to help. It also will speed development if another datetime is found in the dash.